src/Npgsql/Internal/DataFormat.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Diagnostics;
+
+namespace Npgsql.Internal;
+
+public enum DataFormat : byte
```

Comment:
Written elsewhere, IIRC the PG naming in the protocol docs is "format code" - any reason not to align with that (and also to just have a single short enum as before, casting to and from it)?